discovery/ec2/ec2.go

```diff
@@ -40,21 +40,22 @@ const (
 	ec2Label                  = model.MetaLabelPrefix + "ec2_"
 	ec2LabelAZ                = ec2Label + "availability_zone"
 	ec2LabelArch              = ec2Label + "architecture"
+	ec2LabelIPv6              = ec2Label + "ipv6"
```

Comment:
This isn't very clear, ipv6 what?

Reply:
sure, will update it to ipv6_address

Reply:
We don't provide a list of addresses anywhere, and that's not very useful for scraping as we need a single address. How are you planning on using this? Are there public and private v6 addresses?

Reply:
EC2 instances might not be reachable via private IPv4 addresses (if scraping is done remotely), and might not have public IPv4 addresses.
In my opinion there are no private/public v6 addrs.
Also, i don't have a usecase for this, i wanted to contribute.. and had a look at: #4611

Reply:
We probably need to know a bit more about the use case to figure out how to best represent this, as the idea appears to be using it as the thing to scrape rather than just for niche filtering use cases.

Reply:
sure 👍